### PR TITLE
added file2 and modified file1

### PR DESCRIPTION
**Pull request by khrystynapavlyk**
Added file2 and modified file1.

### CODE REVIEW

**Review by red4ed — APPROVED** [review]
Nice changes!

File: file2.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+fijifjjfd
+kfjkfjjg
+jfjlgk
+2222
```
Comment (red4ed) [inline file2.txt:4]: Interesting.

Comment (red4ed) [reply file2.txt:4]: Hmm.

**Comment by red4ed** [comment]
Wow

File: file2.txt (hunk 1 above)
Comment (khrystynapavlyk) [inline file2.txt:4]: ааа

Comment (red4ed) [reply file2.txt:4]: Wow

**Review by red4ed — COMMENTED** [review]
Good

**Pull request merged**